app/Infrastructure/Controllers/PostOpenWalletController.php

```diff
@@ -3,7 +3,7 @@
 namespace App\Infrastructure\Controllers;
```

Comment:
Do you think this is the best name we could give to the controller? Is the method used in this controller really necessary in the name?